2.6/s2i/bin/usage

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+DISTRO="`grep ^ID= /etc/*-release | cut -d'"' -f2`"
+
+cat <<EOF
+This is a S2I ruby-${RUBY_VERSION} ${DISTRO} base image:
+To use it, install S2I: https://github.com/openshift/source-to-image
+
+Sample invocation:
+
+s2i build https://github.com/sclorg/s2i-ruby-container.git --context-dir=${RUBY_VERSION}/test/puma-test-app/ ${IMAGE_NAME} ruby-sample-app
+
+
+You can then run the resulting image via:
+podman run -p 8080:8080 ruby-sample-app
```

Comment:
The same here. Is it intended? Like replacing docker with podman

Reply:
Same as above.

Note that in case you still use docker, you can have it wrapped podman :) _[there's a package for that]_ or is it the other way around?